ONSdigital/takeon-ui
# Feature/spp3275/collapsible contributor details

Added a collapsible menu for additional contributor details
Also added a new file BaseFormView for the code to allow edit, viewonly and override screens to be shared and changed more easily
Story Link: https://collaborate2.ons.gov.uk/jira/secure/RapidBoard.jspa?rapidView=1583&projectKey=SPP&view=planning&selectedIssue=SPP-3275&issueLimit=100

Test Link: https://dc-ui-spp102.es-take-on.aws.onsdigital.uk/

To test:
1) Choose any survey	
2) Check contributor information above and within the collapsible drop down is correct	
3) Check all 3 pages, view,edit and override for matching and correct data
